shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/main/java/org/apache/shardingsphere/sharding/route/strategy/ShardingStrategyFactory.java

```diff
@@ -43,11 +43,13 @@
      * 
      * @param shardingStrategyConfig sharding strategy configuration
      * @param shardingAlgorithm sharding algorithm
+     * @param defaultShardingColumn default sharding column
      * @return sharding strategy instance
      */
-    public static ShardingStrategy newInstance(final ShardingStrategyConfiguration shardingStrategyConfig, final ShardingAlgorithm shardingAlgorithm) {
+    public static ShardingStrategy newInstance(final ShardingStrategyConfiguration shardingStrategyConfig, final ShardingAlgorithm shardingAlgorithm, final String defaultShardingColumn) {
         if (shardingStrategyConfig instanceof StandardShardingStrategyConfiguration && shardingAlgorithm instanceof StandardShardingAlgorithm) {
-            return new StandardShardingStrategy(((StandardShardingStrategyConfiguration) shardingStrategyConfig).getShardingColumn(), (StandardShardingAlgorithm) shardingAlgorithm);
+            return new StandardShardingStrategy(((StandardShardingStrategyConfiguration) shardingStrategyConfig).getShardingColumn() == null
```

Comment:
@tuichenchuxin `((StandardShardingStrategyConfiguration) shardingStrategyConfig).getShardingColumn() == null` may be better.